es-core/src/components/TextComponent.cpp

```diff
@@ -158,11 +158,9 @@ void TextComponent::calculateExtent()
 	if(mAutoCalcExtent.x())
 	{
 		mSize = mFont->sizeText(mUppercase ? Utils::String::toUpper(mText) : mText, mLineSpacing);
-	}else{
-		if(mAutoCalcExtent.y())
-		{
-			mSize[1] = mFont->sizeWrappedText(mUppercase ? Utils::String::toUpper(mText) : mText, getSize().x(), mLineSpacing).y();
-		}
+	}else if(mAutoCalcExtent.y())
+	{
+		mSize.y() = mFont->sizeWrappedText(mUppercase ? Utils::String::toUpper(mText) : mText, getSize().x(), mLineSpacing).y();
```

Comment:
Apologies, maybe I'm not fully familiar with these semantics, but why did we change the left side of the assignment to a function (changed [1] to .y() ) and more importantly, does it actually work?